app/javascript/components/ListComment.jsx

```
@@ -17,7 +17,17 @@ export default function ListComment({ report }) {
               <div className="card-list-item__user-icons">
                 {report.comments.map((comment) => {
                   return (
-                    <CommentUserIcon comment={comment} key={comment.user_id} />
+                    <a
+                      className="card-list-item__user-icons-icon"
+                      href={`/users/${comment.user_id}`}>
+                      <UserRoleStatusSpan
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)